presto-docs/src/main/sphinx/release/release-0.280.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.280
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where if a query was submitted before the resource group configuration had been loaded, all queries to the cluster would fail with a ``NullPointerException``. This fixes a race condition where a query can start constructing resource groups before the ``ResourceGroupConfigurationManager`` is loaded. 
```

Comment:
I would mostly drop the last sentence and change this to:
```
Fix a race condition, where if a query was submitted before the resource group configuration had been loaded by the Presto server, all queries to the cluster would fail with a ``NullPointerException``. 
```

Reply:
All 4 from line# 17 to 20 are not usable or just line# 20? Can you please confirm?